.github/actions/branch-tag/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -l
+
+BRANCH_NAME="${1#refs/heads/}"
+
+# shellcheck disable=SC2039
+BRANCH_TAG="${BRANCH_NAME/\//-}"
+
+echo "::set-env name=BRANCH_TAG::$BRANCH_TAG"
```

Comment:
Why using `set-env` over `export BRANCH_TAG=$BRANCH_TAG`?

Reply:
The commands on steps are written as different scripts and run by different shells, so the environment variables will NOT persist between them.
https://www.edwardthomson.com/blog/github_actions_15_sharing_data_between_steps.html

So I used the `set-env` command to set it
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#about-workflow-commands